scripts/release-checks/dependency-ranges/build-angular-peer-deps-check.mts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import path from 'path';
+import url from 'url';
+import semver from 'semver';
+import { loadPackageJson } from './pkg-json.mjs';
+
+/** Path to the current directory. */
+const currentDir = path.dirname(url.fileURLToPath(import.meta.url));
+
+/** Path to the project directory. */
+const projectDir = path.join(currentDir, '../../../');
+
+export async function checkBuildAngularPeerDependencies(
+  newVersion: semver.SemVer,
+): Promise<string[]> {
+  const pkgJson = await loadPackageJson(
+    path.join(projectDir, 'packages/angular_devkit/build_angular/package.json'),
```

Comment:
`packages/ngtools/webpack/package.json` also have a peer dep that need to be checked. 

Instead of listing the packages here, I suggest to use the default export from `'lib/packages.js'

```
import packages from '../lib/packages.js';
```

The above, has already the loaded package.json files, and paths.

Reply:
Changed to checking _all_ packages now, using `lib/packages.js`